NCEP/run_graphcast.py

```diff
@@ -179,9 +180,20 @@ def get_predictions(self):
         self.save_grib2(forecasts)
 
     def save_grib2(self, forecasts):
+        converter = Netcdf2Grib()
+
+        # Call and save f000 in grib2
+        ds = self.current_batch
+        ds = ds.drop_vars(['geopotential_at_surface','land_sea_mask', 'total_precipitation_6hr'])
```

Comment:
Can we only drop total_precipitation_6hr and keep hgtsfc and land sea mask?

Reply:
Those are not in the forecasts grib2 file, do you still need them in the f000 file?

Reply:
I see. Let's drop them then to be consistent with other forecasts.